pkg/object/httpserver/runtime.go

```diff
@@ -508,6 +531,86 @@ func (r *runtime) newMetrics(name string) *metrics {
 				Objectives: prometheushelper.DefaultObjectives(),
 			},
 			httpserverLabels).MustCurryWith(commonLabels),
+		M1: prometheushelper.NewGauge(
+			"httpserver_m1",
+			"the m1 of the http server",
+			httpserverLabels[:5]).MustCurryWith(commonLabels),
+		M5: prometheushelper.NewGauge(
+			"httpserver_m5",
+			"the m5 of the http server",
+			httpserverLabels[:5]).MustCurryWith(commonLabels),
+		M15: prometheushelper.NewGauge(
+			"httpserver_m15",
+			"the m15 of the http server",
+			httpserverLabels[:5]).MustCurryWith(commonLabels),
+		M1Err: prometheushelper.NewGauge(
+			"httpserver_m1_err",
+			"the m1 error of the http server",
+			httpserverLabels[:5]).MustCurryWith(commonLabels),
+		M5Err: prometheushelper.NewGauge(
+			"httpserver_m5_err",
+			"the m5 error of the http server",
+			httpserverLabels[:5]).MustCurryWith(commonLabels),
+		M15Err: prometheushelper.NewGauge(
+			"httpserver_m15_err",
+			"the m15 error of the http server",
+			httpserverLabels[:5]).MustCurryWith(commonLabels),
+		M1ErrPercent: prometheushelper.NewGauge(
+			"httpserver_m1_err_percent",
+			"the m1 error percent of the http server",
+			httpserverLabels[:5]).MustCurryWith(commonLabels),
+		M5ErrPercent: prometheushelper.NewGauge(
+			"httpserver_m5_err_percent",
+			"the m5 error percent of the http server",
+			httpserverLabels[:5]).MustCurryWith(commonLabels),
+		M15ErrPercent: prometheushelper.NewGauge(
+			"httpserver_m15_err_percent",
+			"the m15 error percent of the http server",
+			httpserverLabels[:5]).MustCurryWith(commonLabels),
+		Min: prometheushelper.NewGauge(
+			"httpserver_min",
+			"the min of the http server",
```

Comment:
please make a little improvement to this, as user can not know it is the minimal value of what.
please also fix the other similiar cases.